torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -91,6 +91,15 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
+        prim::StringLiteral,
+        [](Node* node) {
+          return [](Stack& stack) {
+            stack.push_back(at::Tensor());
```

Comment:
I'd rather not have this dummy tensor output. It would be better to have a string lowering pass, where we gather all `StringLiteral` inputs to `print`s in its `std::vector<std::string>` attribute, and remove them from the graph entirely (kind of like what you do in the `prim::Print` node, but earlier)